pkg/subctl/cmd/deploybroker.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/submariner-operator/pkg/broker"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/datafile"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/operator/install"
+)
+
+var disableDataplane bool
+
+func init() {
+	deployBroker.PersistentFlags().BoolVarP(&disableDataplane, "no-dataplane", "n", false,
+		"Don't install the submariner dataplane on the broker")
+	rootCmd.AddCommand(deployBroker)
+}
+
+const IPSECPSKBytes = 48 // using base64 this results on a 64 character password
```

Comment:
We use a mix of 32 and 48 for this currently I think.

Reply:
That's from the other patch. You mean testing vs implementation? , I'll check that on a separate PR, thanks daniel.